File: javascript/fibonacci.js
/**
 * Fibonacci Algorithm in JavaScript
 * 
 * Author: Elton Fonseca
 */

 function fibonacci(term)
 {
     fib0 = fib1 = 1;

     console.log(fib0);
     console.log(fib1);
     
     for(i = 3; i <= term; i++)
     {
        aux = fib0 + fib1;
        fib0 = fib1;
        fib1 = aux;

        console.log(fib1);
     }
 }
 fibonacci(10);

File: python/fibonacci.py
#   Fibonacci Algorithm in Python
#
# Author: Elton Fonseca
#

def fibonacci(term):
    fib0 = fib1 = 1
    print fib0
    print fib1
    for i in range(3, term + 1):
        aux = fib0 + fib1
        fib0 = fib1
        fib1 = aux
        print fib1

fibonacci(10)